docs/articles/architecture/arch-glossary.rst

```diff
@@ -5,10 +5,22 @@
 Glossary
 ========
 
-+---------+-------------------------------------------+
-| Term    | Definition                                |
-+=========+===========================================+
-| Service | A separate process performing a function. |
-+---------+-------------------------------------------+
-| SOTA    | Software-Over-The-Air                     | 
-+---------+-------------------------------------------+
++------------+------------------------------------------------+
+| Term       | Definition                                     |
++============+================================================+
+| Service    | A separate process performing a function.      |
++------------+------------------------------------------------+
+| SOTA       | Software-Over-The-Air                          |
++------------+------------------------------------------------+
+| Bootloader | A component in charge of loading a kernel.     |
++------------+------------------------------------------------+
+| Kernel     | Refers to the Linux kernel in this document.   |
++------------+------------------------------------------------+
+| Rootfs     | Root File System (main partition mounted on /) |
++------------+------------------------------------------------+
+| ECU        | Electronic Control Unit                        |
++------------+------------------------------------------------+
+| TPM        | Trusted Platform Modules                       |
++------------+------------------------------------------------+
+| TEE        | ARM Trusted Execution Environment              |
++------------+------------------------------------------------+
```

Comment:
Please have a look at the `:glossary:` and `:term:` directives available in sphinx.

Reply:
+1 it's my fault we've created the glossary in this way. Sorry!